chain-abstraction/sol-implementation/assertion_chain.go

```diff
@@ -945,12 +953,12 @@ func (a *AssertionChain) AssertionUnrivaledBlocks(ctx context.Context, assertion
 	return prevNode.SecondChildBlock - prevNode.FirstChildBlock, nil
 }
 
-// GetAssertionCreationBlock returns parent chain block number when the assertion was created.
+// GetAssertionCreationL1Block returns parent chain block number when the assertion was created.
```

Comment:
Shouldn't this just be `GetAssertionCreationParentBlock`

It's only `L1` when the current chain is an `L2`, right?

Reply:
Done.